numba/core/inline_closurecall.py

```diff
@@ -751,9 +768,8 @@ def stararg_handler(index, param, default):
                               x in default_tuple.items]
                 args = args + const_vals
             else:
-                raise NotImplementedError(
-                    "Unsupported defaults to make_function: {}".format(
-                        defaults))
+                msg = "Unsupported defaults to make_function: {}"
+                raise NotImplementedError(msg.format(callee_defaults))
```

Comment:
```suggestion
                raise NotImplementedError(msg.format(defaults))
```
? Is the issue that `defaults` doesn't exist or is this a mistake?

Reply:
There is, as far as I can tell, no variable in this context called `defaults`. I presumed it was a typo, because `callee_defaults` is actually either `callee.defaults` or `callee.__defaults__`, which were the only things that could concievably be meant as "defaults", and the `if` statement is looking at `callee_defaults`. See the following elided snippet:
```python
if callee_defaults:
    debug_print("defaults = ", callee_defaults)
    if isinstance(callee_defaults, tuple):  # Python 3.5
        pass # Elided, this is just for GitHub syntax highlighting.
    elif (isinstance(callee_defaults, ir.Var)
                    or isinstance(callee_defaults, str)):
        pass # Elided, this is just for GitHub syntax highlighting.
    else:
        raise NotImplementedError(
            "Unsupported defaults to make_function: {}".format(
                defaults))
```
I'm given further confidence that this was the intended meaning by the log message at the top - printing "defaults =" then `callee_defaults` would suggest that an "Unsupported defaults" would also refer to `callee_defaults`.